forte_wrapper/elastic/elastic_search_processor.py

```diff
@@ -43,12 +43,28 @@ def initialize(self, resources: Resources, configs: Config):
 
     @classmethod
     def default_configs(cls) -> Dict[str, Any]:
+        """
+        This defines a basic config structure for ElasticSearchProcessor.
+        :return: A dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
+            - query_pack_name: the query pack's name.
+            - index_config: the ElasticSearchIndexer's config.
+            - field: Field name that will be used when creating the new
+            datapack.
+            - response_pack_name_prefix: the pack name prefix to be used
+            in response datapacks.
+            - only_pass_content: boolean, defines if only the content field
```

Comment:
Now I understand this variable, but I suggest this change. I also suggest changing the name of this config field, because `content` is not a name known by the user. The default value for `field` is `content`, but that can be changed.

> - indexed_text_only: boolean, defines whether the returned value from the field (as specified by the `field` configuration) will be considered as plain text.  If True, a new data pack will be created and the value will be used as the `text` for the data pack. Otherwise, the returned value will be considered as serialized data pack, and the returned data pack will be created by deserialization. Default is True.

Reply:
I agree, that seems make more sense. I have updated the param again to make it work.